pkg/clients/aws.go

```diff
@@ -633,9 +628,14 @@ func CleanError(err error) error {
 	if err == nil {
 		return err
 	}
-	if awsErr, ok := err.(awserr.Error); ok {
-		return awserr.New(awsErr.Code(), awsErr.Message(), nil)
-	}
+	// TODO cvodak revisit this
```

Comment:
Would you be able to test whether we need this for this v2 SDK? The reason we have it is to remove the request id from the error so that we don't get re-queued immediately since there is a change at every reconcile. If v2 errors don't contain a unique request id, then it might be fine not to have any handling here.